addons/mail/static/src/discuss/call/common/rtc_session_model.js

```diff
@@ -110,6 +110,16 @@ export class RtcSession extends Record {
         compute() {
             return this.is_screen_sharing_on || this.is_camera_on;
         },
+        /** @this {import("models").RtcSession} */
+        onUpdate() {
+            if (
+                this.isVideoStreaming &&
+                this.store.rtc.selfSession?.in(this.channel?.rtc_session_ids) &&
+                !this.channel.activeRtcSession
+            ) {
+                this.channel.focusAvailableVideo();
+            }
+        },
```

Comment:
Specs say only calls in direct chat should have this behavior.
When discuss calls were introduced, this behavior was applied to all conversations and this was not great. So better limit to 1-to-1 chat as written the specs.